Tribler/Core/defaults.py

```diff
@@ -163,6 +164,26 @@
 sessdefaults['http_api']['enabled'] = False
 sessdefaults['http_api']['port'] = -1
 
+# Credit mining config
+sessdefaults['credit_mining'] = OrderedDict()
+sessdefaults['credit_mining']['enabled'] = True
```

Comment:
Any new feature should be disabled by default until we are sure it's fine to have it on by default. This one is experimental so disabled by default.